server/server.js

```diff
@@ -163,6 +163,7 @@ updatereq = true;
     });
 
     socket.on('disconnect', function () {
+    	if(findPlayer(userID) !== null){
```

Comment:
Should probably save the result from findPlayer(userId) in a var before the if statement. Since now it'll look up the user twice. See line 167 also.